system/Config/ForeignCharacters.php

```diff
@@ -18,6 +18,8 @@ class ForeignCharacters
 {
     /**
      * Without further ado, the list of foreign characters.
```

Comment:
While you're here, please remove this useless comment.

Reply:
Done.